runtime/store.go

```diff
@@ -291,3 +291,8 @@ func (s *Store) Update(updater interface{}) {
 		s.errorHandler(err)
 	}
 }
+
+// Get returns the value of the data that's kept in the current state / Store
+func (s *Store) Get() interface{} {
+	return s.data
```

Comment:
This will return a `reflect.Value`. I suspect you wanted to return `s.data.Interface()`?

Reply:
Ah very true.